exist-core/src/main/java/org/exist/xquery/LocationStep.java

```diff
@@ -227,7 +227,7 @@ private Sequence processPredicate(Sequence outerSequence, final Sequence context
             result = pred.evalPredicate(outerSequence, result, axis);
             // subsequent predicates operate on the result of the previous one
             outerSequence = null;
-            context.setContextSequencePosition(-1, null);
```

Comment:
Ah! I figured out there was 'something' starting from -1 ....
Did you run the xqts stuff?

Reply:
No, I haven't

Reply:
Do you expect that this would bring us closer to 100%?

Reply:
OK, I installed the SNAPSHOT then packaged XQTS-runner and ran that.
It hangs at
`03 Mar 2021 11:17:13,313 [XQTSRunnerSystem-akka.actor.default-dispatcher-8] INFO  (slf4j.scala [info]:159) - Serialized results for TestSet: prod-Predicate OK.`
I waited half an hour ... twice. So, no XQTS run :(

Reply:
Just checked, this is with the lates changes to XQTS-runner `8e39a3dd824d05a55d931b1624b18e83491264c0`

Reply:
@line-o I would need more information about the XQTS problem to be able to help... For example, what arguments did you give to XQTS? and what was the stack trace / log when you say it produced no results. Did it hang or did it quit? Maybe open an issue against XQTS with steps to reproduce what you are seeing?

Reply:
in exist-db branch fix/3737

1.    mvn clean install -DskipTests=true

in xqts-runner branch master

1. sbt package
2. target/scala-2.13/exist-xqts-runner-assembly-1.0.0.jar -l tests -x HEAD

Reply:
@line-o addressed here - https://github.com/eXist-db/exist/pull/3774

Reply:
Works now 
NOTE: I had to use `sbt assemble` instead of `sbt package`